pkg/work/webhook/v1alpha1/manifestworkreplicaset_validating.go

```diff
@@ -78,7 +78,7 @@ func validatePlaceManifests(mwrSet *workv1alpha1.ManifestWorkReplicaSet) error {
 }
 
 func checkFeatureEnabled() error {
-	if !utilfeature.DefaultMutableFeatureGate.Enabled(ocmfeature.ManifestWorkReplicaSet) {
+	if !features.HubMutableFeatureGate.Enabled(ocmfeature.ManifestWorkReplicaSet) {
```

Comment:
nit: because we already checked if the feature is enabled https://github.com/open-cluster-management-io/ocm/pull/673/files#diff-d26f37502c2c85732485a395caad60404fad094e6eac48fe2df438b2f24871d1R78 , do we still need to check again here?